### ISSUE TITLE
Printing shelfmarks in bibliography corresp

### ISSUE THREAD

**Issue by nafisa-valieva** [SChAth/ScAthiop#2695]
Hello! I am sorry in advance for any possible mistakes in formulating the issue. I have a question in connection to [[https://github.com/BetaMasaheft/Documentation/issues/2690#issuecomment-2476360878](https://github.com/BetaMasaheft/Documentation/issues/2690)], but from a different perspective. Henok was edited and translated several times (Erho´s article counts 7 lines). I have checked the guidelines, but could not find an example fitting this case. At the moment LIT1340EnochE  contains Geez text of undefined origin and no ListWit at all. I would like to encode information about manuscripts used for various editions and translations as clearly as possible, but for the moment I am not sure how. I hope someone can point me to the right example / suggest a good strategy or just tell me that I should wait till there is a solution. 
Thank you very much in advance!

**Comment by eu-genia** [comment]
The association of bibliography and editions is done by providing `xml:id` for each witness and linking editions and texts to these sigla in `corresp`. This is why witnesses used in an edition must have an `xml:id` as the guidelines say.

see e.g.  `LIT1709Kebran`
which has

```
<listWit rend="edition">
                        <witness corresp="BNFet5" xml:id="bnfet5">BnF éthiopien 5</witness>
                       <witness corresp="BNFet146" xml:id="bnfet146">BnF éthiopien 146</witness>
                       <witness corresp="BNFabb97" xml:id="bnfabb97">BnF d'Abbadie 97</witness>
                       <witness corresp="BNFabb132" xml:id="bnfabb132">BnF d'Abbadie 132</witness>
                       <witness corresp="BNFabb152" xml:id="bnfabb152">BnF d'Abbadie 152</witness>
                        <witness corresp="BLorient818" xml:id="blor818">BL oriental 818</witness>
                       <witness corresp="BLorient819" xml:id="blor819">BL oriental 819</witness>
                        <witness corresp="EMML50" xml:id="emml50">EMML 50</witness>
                        <witness corresp="ANLcr27" xml:id="anlcr27">ANL Conti Rossini 27</witness>
                        <witness corresp="BDLbruce87" xml:id="bdlbruce87">Bodleian Bruce 87</witness>
                        <witness corresp="BDLbruce93" xml:id="bdlbruce93">Bodleian Bruce 93</witness>
                       <witness corresp="BerOrFol395" xml:id="berlin395">Berlin, SPK or. fol. 395</witness>
                    </listWit>
```
and then e.g. 

```
<listBibl type="editions">
                 ...
                 <bibl corresp="#bnfet5 #bnfet146 #bdlbruce93 #bdlbruce87 #berlin395 #blor818 #blor819">
                    <ptr target="bm:Bezold1905KebraN"/>
                 </bibl>
              </listBibl>
```


(PS for a good example of an edition (though not multiple, but tecnically it is the same, we can have as many as needed) see `LIT4916PhysB `where Massimo edited chapter 4. As far as I can see the apparatus is visualized nicely.)

**Comment by eu-genia** [comment]
https://betamasaheft.eu/Guidelines/?id=bibliographic-references

**Comment by nafisa-valieva** [comment]
Thank you very much! it is very helpful! I am not sure whether it is a problem of encoding or visualisation, in case it is a matter of visualisation, would it be possible not to have ´about´ in the Editions Bibliography (about: witness , [Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://lit1709kebran/#bnfet5)[Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://betamasaheft.eu/LIT1709Kebran#bnfet146)[Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://betamasaheft.eu/LIT1709Kebran#bdlbruce93)[Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://betamasaheft.eu/LIT1709Kebran#bdlbruce87)[Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://betamasaheft.eu/LIT1709Kebran#berlin395)[Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://betamasaheft.eu/LIT1709Kebran#blor818)[Kǝbra nagaśt, Paris, Bibliothèque nationale de France, BnF Éthiopien 5](https://betamasaheft.eu/LIT1709Kebran#blor819))?

**Comment by eu-genia** [comment]
yes, I can modify the way the shelfmark is printed, but since this is not a true bug and just a question of aesthetics I would for the moment put it on the wishlist and wait for the new release if that is OK and you can wait a bit

**Comment by eu-genia** [comment]
Probably for such cases the best workaround would be to not print the shelfmark in full but just the sigla assigned with a link?

**Comment by nafisa-valieva** [comment]
> yes, I can modify the way the shelfmark is printed, but since this is not a true bug and just a question of aesthetics I would for the moment put it on the wishlist and wait for the new release if that is OK and you can wait a bit

thank you! I can wait.

**Comment by nafisa-valieva** [comment]
[https://github.com/BetaMasaheft/Documentation/issues/2793](url)
